updated manual modpack install section (#632)
ldtteam/MinecoloniesWiki: source/installation/java.md

MineColonies doesn't always accept the latest version of Structurize. To ensure you have the required version, first, click on the MineColonies file name (on the Curseforge website). You'll be taken to the changelog, where it should say the accepted Structurize version at the top. Download that exact version.

> #### Cannot Add Modpack Without External Installer.

Reviewer: ```suggestion
#### Cannot Add Modpack Without External Installer
```

> #### If Adding Modpack you will require an external installer.

Reviewer: Title case here, since it's a heading. Maybe say "Cannot Add Modpack Without External Installer"

> If you are adding the entire modpack to be able to play on the official server, you will require an external launcher to download the pack. The [Curseforge launcher](https://curseforge.overwolf.com/) works on Windows or macOS. Other options include [GDlauncher](https://gdevs.io/#downloadContainer) or [MultiMC](https://multimc.org/#Download), both offering Windows, macOS and Linux support. Finally if you can not or do not wish to 'install' a launcher you can use [CMPDL](https://github.com/Vazkii/CMPDL) that will run a small java program to download the required modpack. 

Reviewer: ```suggestion
If you are adding the entire modpack to be able to play on the official server, you will require an external launcher to download the pack. The [Curseforge launcher](../../source/installation/curseforge) works on Windows or macOS. Other options include [GDlauncher](../../source/installation/gdlauncher) or [MultiMC](https://multimc.org/#Download), both offering Windows, macOS and Linux support. Finally, if you can not or do not wish to 'install' a launcher you can use [CMPDL](https://github.com/Vazkii/CMPDL) that will run a small Java program to download the required modpack. 
```
Probably better to link to the wiki page instead of the direct link

> Unfortunately manual 'installing' modpacks is no longer an option as Curseforge no longer distributes mods in the downloaded zip files for modpacks due to licensing issues.

Reviewer: ```suggestion
Unfortunately, manually 'installing' modpacks is no longer an option as Curseforge no longer distributes mods in the downloaded zip files for modpacks due to licensing issues.
```
